src/components/inline_editable_renderer/inline_editable_renderer.tsx

```diff
@@ -1,11 +1,12 @@
 import { Colors } from '@blueprintjs/core';
 import styled from '@emotion/styled';
-import type { ReactNode } from 'react';
+import type { KeyboardEvent, ReactNode } from 'react';
 import { useCallback, useMemo, useState } from 'react';
 
 export interface InlineRendererEditableProps<T extends HTMLElement> {
   ref: (node: T | null) => void;
-  toggle: () => void;
+  stopRendering: () => void;
```

Comment:
`stopRendering` is little weird, I suggest `exit`

And add a jsdoc to explain what it does.